includes/forms/template.php

```diff
@@ -1411,7 +1411,7 @@ function give_get_login_fields( $form_id ) {
 		 *
 		 * @param int $form_id The form ID.
 		 */
-		// do_action( 'give_checkout_login_fields_before', $form_id );
+		do_action( 'give_checkout_login_fields_before', $form_id );
```

Comment:
@Sidsector9 and @ravinderk note I have renamed this action and the after action to:

`give_donation_form_login_fields_before` and `give_donation_form_login_fields_after`.